dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESMappingUtilHelper.java

```diff
@@ -132,6 +155,25 @@ private void addCustomMappingForRelationships(final String indexName,
         }
     }
 
+    /**
+     * Creates a json mapping for a relationship and saves it into the specified index
+     * @param indexName - Index where the mapping will be saved (Full index name including cluster id)
+     * @param relationshipName - Relationship to be indexed
+     * @throws JSONException
+     * @throws IOException
+     */
+    private void putRelationshipMapping(final String indexName, final String relationshipName)
+            throws JSONException, IOException {
+        final JSONObject properties = new JSONObject();
+        properties.put("properties", new JSONObject()
+                .put(relationshipName,
+                        new JSONObject("{\n"
```

Comment:
I wonder if we should not use something such as 
```
JSONObjectBuilder()
    .add("type","keyword")
   .add("ignore_above",8191).build()
```

Reply:
It seems the JSONObject has already methods to do it. Not need to create any builder